design-documents/graph-ql/coverage/customer-orders.md

```diff
@@ -325,6 +325,30 @@ type CommentItem {
 }
 ```
 
+## OrderAddress type
+```graphql
+
+type OrderAddress @doc(description: "OrderAddress contains detailed information about the order billing and shipping addresses"){
+    firstname: String! @doc(description: "The first name of the person associated with the shipping/billing address")
+    lastname: String! @doc(description: "The family name of the person associated with the shipping/billing address")
+    middlename: String @doc(description: "The middle name of the person associated with the shipping/billing address")
+    region: String @doc(description: "The state or province name")
+    region_id: ID @doc(description: "The unique ID for a pre-defined region")
+    country_code: CountryCodeEnum @doc(description: "The customer's order country")
+    street: [String]! @doc(description: "An array of strings that define the street number and name")
```

Comment:
Let's make `street` a non-nullable list as well:

```graphql
street: [String!]
```

That just makes sure a `null` can't be hiding in the list of street items.